sources/Service/Notification/Event/EventiTopNotificationGC.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace  Combodo\iTop\Service\Notification\Event;
+
+use DBObjectSearch;
+use DBObjectSet;
+use Exception;
+use ExceptionLog;
+use iBackgroundProcess;
+use MetaModel;
+
+class EventiTopNotificationGC implements iBackgroundProcess
+{
+
+	public function Process($iUnixTimeLimit)
+	{
+		try {
+			$sDeletionTime = MetaModel::GetConfig()->Get('notifications.itop.read_notification_retention');
+			$oDBObjectSearch = DBObjectSearch::FromOQL("SELECT EventiTopNotification WHERE read='yes' AND read_date < DATE_SUB(NOW(), INTERVAL :deletion_time DAY)", ['deletion_time' => $sDeletionTime]);
```

Comment:
```suggestion
			$iDeletionTime = (int) MetaModel::GetConfig()->Get('notifications.itop.read_notification_retention');
			$oDBObjectSearch = DBObjectSearch::FromOQL("SELECT EventiTopNotification WHERE read='yes' AND read_date < DATE_SUB(NOW(), INTERVAL :deletion_time DAY)", ['deletion_time' => $iDeletionTime]);
```

So we know what we are manipulating.